internal/datanode/index/task_index.go

```diff
@@ -217,29 +221,29 @@ func (it *indexBuildTask) Execute(ctx context.Context) error {
 	var fieldDataSize uint64
 	if vecindexmgr.GetVecIndexMgrInstance().IsDiskANN(indexType) {
 		// check index node support disk index
-		if !Params.IndexNodeCfg.EnableDisk.GetAsBool() {
-			log.Warn("IndexNode don't support build disk index",
+		if !paramtable.Get().DataNodeCfg.EnableDisk.GetAsBool() {
```

Comment:
just remove this check?

Reply:
We will discuss and decide in the next PR whether to remove it.